standard/sections/clause_7_normative_text.adoc

```diff
@@ -884,6 +949,14 @@ include::../recommendations/core/REC_distribution.adoc[]
 }]
 ----
 
+include::../requirements/core/REQ_links.adoc[]
+include::../recommendations/core/REC_links.adoc[]
+include::../recommendations/core/PER_links.adoc[]
+
+===== Distribution information
```

Comment:
I think it will also be useful to have a summary table of properties for distribution info...

Reply:
Distribution is an optional element with detailed requirements.  We can cover post INFCOM-3.